antlr/YorkieSchemaParser.ts

```
@@ -0,0 +1,2112 @@
+
+
+export class DeclarationContext extends ParserRuleContext {
+	public typeAliasDeclaration(): TypeAliasDeclarationContext | undefined {
+		return this.tryGetRuleContext(0, TypeAliasDeclarationContext);
+	}
+	public variableDeclaration(): VariableDeclarationContext | undefined {
+		return this.tryGetRuleContext(0, VariableDeclarationContext);
+	}
+	constructor(parent: ParserRuleContext | undefined, invokingState: number) {
+		super(parent, invokingState);
+	}
```

Biome

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)